redisinsight/ui/src/slices/interfaces/rdi.ts

```diff
@@ -1,11 +1,19 @@
 import { Nullable } from 'uiSrc/utils'
 import { Rdi as RdiInstanceResponse } from 'apiSrc/modules/rdi/models/rdi'
+import {PageNames} from "uiSrc/constants";
```

Comment:
<img width="825" alt="Screenshot 2024-03-01 at 3 30 49 PM" src="https://github.com/RedisInsight/RedisInsight/assets/112435081/bad36da3-1b5c-45e2-8e67-cd2e3c493518">

Reply:
Thank you. Fixed